libs/sdk-ui-dashboard/src/model/events/alerts.ts

```diff
@@ -0,0 +1,202 @@
+// (C) 2021 GoodData Corporation
+import { IWidgetAlert, IWidgetAlertDefinition } from "@gooddata/sdk-backend-spi";
+import { IDashboardEvent } from "./base";
+import { DashboardContext } from "../types/commonTypes";
+
+/**
+ * This event is emitted after the Kpi alert is successfully saved.
+ *
+ * @internal
+ */
+export interface DashboardAlertCreateSuccess extends IDashboardEvent {
```

Comment:
events describe what happened. they need to be in past tense. `DashboardAlertCreated` etc.